CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ↗️ = updated since previous release
 
+
+## v2025.8.14
```

Comment:
Change to "Unreleased" for now  – we can add the new date as part of the release process.